### ISSUE TITLE
Ruby bindings via C FFI

### ISSUE THREAD

**Issue by thedlop** [ergoplatform/sigma-rust#575]
I'm currently working on this. Figured I'd mention it. Curious if it is worth a bounty. Thank you!

**Comment by greenhat** [comment]
Thank you! Ruby was not on my radar initially. We believe we don't have Ruby experts to maintain it. Do you plan to host it in a separate repo?

**Comment by thedlop** [comment]
Yes, I was planning to initially release it as a public repo on github and as a ruby gem (ruby package) so it could be easily downloaded. And opening a PR here to include it under `/bindings/` like all the others. I'm happy to do whatever. 

Understandable ya'll don't wish to maintain it if no one knows Ruby. I'm happy to maintain it for now.

**Comment by greenhat** [comment]
Sounds good! You might want to consider participating in the ErgoHack hackathon with this idea. It's just started and will be going until the end of May. Check out our discord - https://discord.gg/kj7s7nb

**Comment by thedlop** [comment]
Will do! I didn't realize I could work on something I already started.

**Comment by thedlop** [comment]
Ok bindings are now on par with iOS bindings aside from not implementing the two or three REST API methods. I  just made the repo public: https://github.com/thedlop/sigma_rb . I still need to package up into a gem so it can be used easily, add documentation, and finish the README.

**Comment by thedlop** [comment]
We can add the gem version badge now: https://badge.fury.io/for/rb/sigma_rb
